js/src/carousel.js

```diff
@@ -283,14 +283,11 @@ class Carousel {
       if (this._pointerEvent && (event.originalEvent.pointerType === PointerType.TOUCH || event.originalEvent.pointerType === PointerType.PEN)) {
         this.touchStartX = event.originalEvent.clientX
       } else if (!this._pointerEvent) {
-        event.preventDefault()
         this.touchStartX = event.originalEvent.touches[0].clientX
       }
     }
 
     const move = (event) => {
-      event.preventDefault()
```

Comment:
I don't understand why do you remove this line ? 🤔
This function `move` is used only for device which do not handle PointerEvents and prevent the "go back" behaviour

Reply:
`preventDefault` on `touchstart` and `touchmove` prevents scrolling (one of the browser's "default actions") in touch-events-only browsers (e.g. iOS Safari). leaving this in would still result in carousels not allowing scrolling on iPhone etc.